### PR TITLE
Added Iraqi Channels, Logos, Updated Streams

### PR DESCRIPTION
**Pull request by themak1985**

### CODE REVIEW

File: channels/iq.m3u [hunk 1]
```diff
@@ -1,49 +1,51 @@
...
 http://media.smc-host.com:1935/redirect/alfadytv.tv/alfadytv.smil?type=m3u8
-#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Arabic" tvg-logo="" group-title="General",Al Fallujah
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Arabic" tvg-logo="http://www.alfallujah.tv/sites/all/themes/progressive/progressive_sub/logo.png" group-title="General",Al Fallujah
 https://player-api.new.livestream.com/accounts/4671269/events/4213685/live.m3u8
-#EXTINF:-1 tvg-id="Al Hurra Iraq ARB" tvg-name="Al Hurra Iraq ARB" tvg-language="Arabic" tvg-logo="" group-title="News",Al Hurra Iraq
+#EXTINF:-1 tvg-id="Al Hurra Iraq ARB" tvg-name="Al Hurra Iraq ARB" tvg-language="Arabic" tvg-logo="https://www.alhurra.com/Content/responsive/MBN/ar-ALH/img/logo.svg" group-title="News",Al Hurra Iraq
 http://mbnhls-lh.akamaihd.net/i/MBN_1@118619/master.m3u8
-#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Arabic" tvg-logo="" group-title="General",Al Masara Al Oula
+#EXTINF:-1 tvg-id="" tvg-name="Al-Masar Al-Oula" tvg-language="Arabic" tvg-logo="https://i.imgur.com/npuXX8e.jpg" group-title="General",Al-Masar Al-Oula
```
Comment (freearhey) [inline channels/iq.m3u]: tvg-name is wrong. It should be the same as the display-name from the EPG. If there is no such channel in the EPG, it is better to leave this attribute blank.

File: channels/iq.m3u [hunk 2]
```diff
@@ -1,49 +1,51 @@
...
+#EXTINF:-1 tvg-id="Al Sharqiya ARB" tvg-name="Al Sharqiya ARB" tvg-language="Arabic" tvg-logo="http://alsharqiyatv.info/tv.jpg" group-title="General",Al Sharqiya
 http://ns8.indexforce.com:1935/home/mystream/playlist.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Arabic" tvg-logo="https://i.imgur.com/qNi4rwG.jpg" group-title="News",Al Sharqiya News
 http://ns8.indexforce.com:1935/alsharqiyalive/mystream/playlist.m3u8
-#EXTINF:-1 tvg-id="Alsumaria ARB" tvg-name="Alsumaria ARB" tvg-language="Arabic" tvg-logo="" group-title="General",Al Sumaria
-https://cllive.itworkscdn.net/alsumariatvlive/token=nva=1574938262~dirs=1~hash=01feee69070f922b8f362/sumariamob.smil/playlist.m3u8
-#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Arabic" tvg-logo="" group-title="General",Al Taghayer
+#EXTINF:-1 tvg-id="Alsumaria ARB" tvg-name="Alsumaria ARB" tvg-language="Arabic" tvg-logo="https://www.alsumaria.tv/uploadImages/ProgramsImages/Channels-LW-1-636916940695846700.png" group-title="General",Al Sumaria
+https://cllive.itworkscdn.net/alsumariatvlive/token=nva=1575257386~dirs=1~hash=0ca680b6400133ce1001b/sumariamob.smil/playlist.m3u8
```
Comment (freearhey) [inline channels/iq.m3u]: New link does not work. Tested in VLC player and streamtest.in

File: channels/iq.m3u [hunk 3]
```diff
@@ -1,49 +1,51 @@
...
 http://cdn.teradek.live:1935/live/astera/chunklist.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Kurdish" tvg-logo="https://i.imgur.com/Kt4dLDE.png" group-title="",AVA Entertainment
 http://95.170.207.28:1935/live/Channel01/chunklist.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Kurdish" tvg-logo="https://i.imgur.com/R0klqgb.jpg" group-title="",AVA TV
 http://51.15.236.137/hls/stream.m3u8
-#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Arabic" tvg-logo="" group-title="General",Babylon TV
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Arabic" tvg-logo="https://babylontv.net/wp-content/themes/babylontv/images/logo.png" group-title="General",Babylon TV
 https://babylonlive.net/babylontv/kardokardo.m3u8
-#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Arabic" tvg-logo="" group-title="General",Dijlah
+#EXTINF:-1 tvg-id="" tvg-name="Dijlah" tvg-language="Arabic" tvg-logo="https://www.dijlah.tv/templates/default-2/images/dlogo.png" group-title="General",Dijlah
```
Comment (freearhey) [inline channels/iq.m3u]: tvg-name is wrong

File: channels/iq.m3u [hunk 4]
```diff
@@ -1,49 +1,51 @@
...
 https://ghaasiflu.online/Dijlah/tracks-v1a1/mono.m3u8
-#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Arabic" tvg-logo="https://i.imgur.com/w5xVJZg.png" group-title="",Dijlah TV
+#EXTINF:-1 tvg-id="" tvg-name="Dijlah" tvg-language="Arabic" tvg-logo="https://i.imgur.com/w5xVJZg.png" group-title="",Dijlah
```
Comment (freearhey) [inline channels/iq.m3u]: tvg-name is wrong

File: channels/iq.m3u [hunk 5]
```diff
@@ -55,10 +57,24 @@ http://live.imamhossaintv.com/live/ih3.m3u8
...
+https://manifest.googlevideo.com/api/manifest/hls_variant/expire/1575254394/ei/GiXkXdzKLMHggwO656Ag/ip/67.81.86.19/id/qXyPOnEcxcw.0/source/yt_live_broadcast/requiressl/yes/hfr/1/ctier/A/maudio/1/hightc/yes/go/1/keepalive/yes/fexp/23842630/dover/11/itag/0/playlist_type/DVR/sparams/expire%2Cei%2Cip%2Cid%2Csource%2Crequiressl%2Chfr%2Cctier%2Cmaudio%2Chightc%2Cgo%2Citag%2Cplaylist_type/sig/ALgxI2wwRQIhAP4xNLjZuf2TOsoRcFBNIbwHgseFPU4FQDutXf7gIUtUAiA6FjCcXuY9afj3bd0N58iiamUZtf7muXRuJIK3z92Fag%3D%3D/file/index.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="Karbala" tvg-language="Arabic" tvg-logo="https://www.karbala-tv.net/s/img/logo-ktv.png" group-title="General",Karbala
+https://player-api.new.livestream.com/accounts/11436227/events/3998452/live.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="Zagros" tvg-language="Arabic" tvg-logo="https://www.zagrosnews.net/sites/default/files/zagros-logo-01_1.png" group-title="General",Zagros
+https://5a3ed7a72ed4b.streamlock.net/zagrostv/SMIL:myStream.smil/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="Türkmeneli" tvg-language="Turkish" tvg-logo="http://www.turkmenelitv.com/images/logotextson.png" group-title="General",Türkmeneli
+http://116.203.71.169/hls/stream1.m3u8
```
Comment (freearhey) [inline channels/iq.m3u]: tvg-name is wrong

File: channels/iq.m3u [hunk 6]
```diff
@@ -55,10 +57,24 @@ http://live.imamhossaintv.com/live/ih3.m3u8
...
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Kurdish" tvg-logo="https://i.imgur.com/GLfhYSH.png" group-title="",Kurdistan TV
 https://5a3ed7a72ed4b.streamlock.net/live/SMIL:myStream.smil/chunklist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="Kurdistan24" tvg-language="Kurdish" tvg-logo="https://www.kurdistan24.net/images/Logo.png" group-title="News",Kurdistan24
+https://k24mediaservice.streaming.mediaservices.windows.net/2d1065bf-4cde-4af3-a425-cef326a13e8c/6a9375c9-7438-4d88-8411-661f162c40d9.ism/manifest
```
Comment (freearhey) [inline channels/iq.m3u]: tvg-name is wrong

File: channels/iq.m3u [hunk 7]
```diff
@@ -77,3 +93,11 @@ https://svs.itworkscdn.net/rudawlive/rudawlive.smil/chunklist.m3u8
...
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Kurdish" tvg-logo="https://i.imgur.com/BcwSPW4.png" group-title="",Speda TV
 http://speda.teradek.live:1935/live/spedahd/chunklist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="GK" tvg-language="Kurdish" tvg-logo="http://www.gksat.tv/wene/logo.png" group-title="News",GK
+http://173.249.45.123:8080/hls/gksat/index.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="WAAR" tvg-language="Kurdish" tvg-logo="http://www.waarmedia.com/wp-content/uploads/2014/12/logo.png" group-title="General",WAAR
+https://5cc309e93b721.streamlock.net/8412/8412/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="Kurd Channel" tvg-language="Kurdish" tvg-logo="http://kurdchannel.live/wp-content/uploads/2019/08/Logo.png" group-title="General",Kurd Channel
+http://116.203.41.89:8080/hls/stream.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="Kurd Channel" tvg-language="Kurdish" tvg-logo="http://kurdchannel.live/wp-content/uploads/2019/08/Logo.png" group-title="General",Kurd Channel
+http://116.203.41.89:8080/hls/stream.m3u8
```
Comment (freearhey) [inline channels/iq.m3u]: tvg-name is wrong

**Review by freearhey — CHANGES_REQUESTED** [review]

**Comment by themak1985** [comment]
I think I messily fixed all the issues @freearhey  - sorry it took 3 commits.

**Review by freearhey — APPROVED** [review]

**Comment by freearhey** [comment]
@themak1985 It's okay.

**Pull request merged**